SignUp/signup.css

```diff
@@ -0,0 +1,5 @@
+.signupContainer{
+    color: aqua;
+    font-weight: bold;
+    font-size: large;
+}
```

Comment:
Bring this Text to the center of the Page

Reply:
done